### ISSUE TITLE
Error in documentation: filter-transform

### ISSUE THREAD

**Issue by Make42** [altair-viz/altair#2434]
The documentation in https://altair-viz.github.io/user_guide/transform/filter.html#user-guide-filter-transform says " here we plot US population distributions for all data except the years 1950-1960" but then uses the code 

```
.transform_filter(
    {'not': alt.FieldRangePredicate(field='year', range=[1900, 1950])}
)
```

which seems to be wrong, when looking at the dates. If this is a mistake, please correct it.

**Comment by jakevdp** [comment]
Thanks! That looks like a typo. Are you interested in submitting a fix?

**Comment by Make42** [comment]
@jakevdp: I do not how to do it without considerable amount of work for me since I am not familiar with altair. I guess I would have to clone the whole thing find out where the error is check all the code etc.?  I just started learning about it two days ago.

**Comment by jakevdp** [comment]
OK, no worries - we'll get it fixed. Thanks for filing the issue!

**Comment by joelostblom** [comment]
This was fixed in #2452
